### ISSUE TITLE
Allow specifying mount directory to --appimage-mount

### ISSUE THREAD

**Issue by crass** [AppImage/AppImageKit#981]
It would be nice (and more script friendly) to be able to specify a mount directory when using `--appimage-mount` rather than getting a random directory from `mkdtemp`.

**Comment by probonopd** [comment]
Can you please give a concrete example of what this would help accomplish? In general, we are trying to keep configurability to a bare minimum in order to reduce complexity and support burden by focusing on defaults that work for most people. Is there something you need to do which you currently cannot do?

**Comment by probonopd** [comment]
Answering my own question: https://github.com/netblue30/firejail/issues/2690#issuecomment-509901803. Let's keep in mind, though, that this may not be a valid option for Firejail anytime soon since there are lots of pre-existing AppImages out there which don't have this functionality.

**Comment by TheAssassin** [comment]
For automation, it'd be nice to be able to choose the mounting directory. Tools like [appimagelint](https://github.com/TheAssassin/appimagelint) which need to inspect the contents of an AppImage have to do some really annoying pipe communication to get the mountpoint directory.

**Comment by crass** [comment]
Right, I originally wrote this because I was thinking it would be useful for firejail, but now I don't think we'll be doing that.  And as you point out, there might not be many appimages that allow that would have this change for a long while, but that's actually an argument for adding this capability as soon as possible for others and potentially firejail down the line if conditions are favorable for going that route later.  As pointed out though, I think this is generally useful, mostly for programs interacting with appimages.

I appreciate not wanting to add complexity to, and in general support that.  I think in this case complexity added will be minimal because the option is already accepted so parsing out an extra `=/some/path` and setting the mountdir var to it is pretty trivial.  So it seems like a its a moderate win for little cost.  And btw, thanks for you guys work on AppImage!

**Comment by TheAssassin** [comment]
It's worth implementing in the next runtime version.
